.github/workflows/linux-ci-helper.sh

```diff
@@ -125,7 +125,7 @@ elif [ "${CONTAINER_FULLNAME}" = "rockylinux:8" ]; then
     # Installing ShellCheck on Rocky Linux is not easy.
     # Give up to run ShellCheck on Rocky Linux as we don't have to run ShellChek on all operating systems.
     #
-    INSTALL_PACKAGES="curl-devel fuse fuse-devel gcc libstdc++-devel gcc-c++ glibc-langpack-en java-11-openjdk-headless libxml2-devel mailcap git automake make openssl-devel attr diffutils curl python3"
+    INSTALL_PACKAGES="curl-devel fuse fuse-devel gcc libstdc++-devel gcc-c++ glibc-langpack-en java-11-openjdk-headless libxml2-devel mailcap git automake make openssl-devel attr diffutils curl-minimal python3"
```

Comment:
I will abandon this change since it appears that we get get curl for free from the base install.  It looks like Rocky Linux 8 does not have curl-minimal but Fedora 35 should:

https://www.rpmfind.net/linux/rpm2html/search.php?query=curl-minimal

Reply:
> I will abandon this change since it appears that we get get curl for free from the base install. It looks like Rocky Linux 8 does not have curl-minimal but Fedora 35 should:
> 
> https://www.rpmfind.net/linux/rpm2html/search.php?query=curl-minimal

Right. I did a bit more of research, and `curl-minimal` is not a `Provides` of `libcurl-minimal` as I thought.

`curl-mininal` provides only the binary and the manual page, and it's available on Fedora and EL9, but not on EL8 including Rocky Linux). `libcurl-minimal` provides the library (which should enough when calling `curl` is not required.

Reply:
This idea was probably not too inspired to begin with since curl is so small.  Spending time figuring out how to package S3Proxy with `jlink` would help remove the biggest dependency:

```
$ rpm -q --queryformat '%10{size} - %-25{name} \t %{arch} \t %{version}\n' libcurl-devel fuse fuse-devel gcc libstdc++-devel gcc-c++ glibc-langpack-en java-11-openjdk-headless libxml2-devel mailcap git automake make openssl-devel curl attr diffutils procps-ng python3-pip | grep -v i686 | sort -n
     82923 - mailcap                     noarch          2.1.53
    127356 - fuse-devel                  x86_64          2.9.9
    135005 - git                         x86_64          2.35.1
    165941 - attr                        x86_64          2.5.1
    220335 - fuse                        x86_64          2.9.9
    723076 - curl                        x86_64          7.79.1
   1010055 - procps-ng                   x86_64          3.3.17
   1438811 - libcurl-devel               x86_64          7.79.1
   1563061 - diffutils                   x86_64          3.8
   1729022 - make                        x86_64          4.3
   1798817 - automake                    noarch          1.16.2
   3629220 - openssl-devel               x86_64          1.1.1l
   5955802 - glibc-langpack-en           x86_64          2.34
   9264370 - libxml2-devel               x86_64          2.9.13
   9402895 - python3-pip                 noarch          21.2.3
  13489200 - libstdc++-devel             x86_64          11.2.1
  30633550 - gcc-c++                     x86_64          11.2.1
  85256329 - gcc                         x86_64          11.2.1
 179192997 - java-11-openjdk-headless    x86_64          11.0.14.1.1
```